### PR TITLE
added CVE-2021-20989 - CVE-2021-20992

### PR DESCRIPTION
**Pull request by CERT-VDE**

### CODE REVIEW

File: 2021/20xxx/CVE-2021-20989.json [hunk 1]
```diff
@@ -1,18 +1,99 @@
 {
```
Comment (cve-team) [inline 2021/20xxx/CVE-2021-20989.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/20xxx/CVE-2021-20989.json:1]: ('Extra data: line 99 column 2 (char 3605)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by CERT-VDE** [comment]
Thank you for the review @cve-team second commit is the fix for the validation error.

**Pull request merged**